### ISSUE TITLE
Filtering headers with None, not working

### ISSUE THREAD

**Issue by serpulga** [wkeeling/selenium-wire#110]
Trying to implement this technique:

```python
driver.header_overrides = {
    'Existing-Header1': None,
    'Existing-Header2': None
}
```

I get this error in a lower lever python module:

```
Traceback (most recent call last):
  File "/Users/serpulga/src/tributi/crwlr/lib/python3.7/site-packages/seleniumwire/proxy/proxy2.py", line 108, in do_GET
    conn.request(self.command, path, req_body, dict(req.headers))
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1293, in _send_request
    self.putheader(hdr, value)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1229, in putheader
    if _is_illegal_header_value(values[i]):
TypeError: expected string or bytes-like object
```

Removing the `header_overrides` assignment makes the issue disappear.

**Python 3.7.6**
**selenium==3.141.0**
**selenium-wire==1.0.12**

**Comment by wkeeling** [comment]
Thanks for raising this.

I have managed to reproduce the issue. It seems that this error will happen if you attempt to filter out a header that does not exist in the request. If the header does exist in the request, then it will be filtered out as expected without an error.

I've now fixed the issue in master so that filtering non-existent headers won't cause an error. That will be available in the next release (out next couple of days). 

You can work around the problem for the time being if you ensure that the headers you wish to filter in `driver.header_overrides` always exist in the request.

Many thanks again for raising the issue.

**Comment by serpulga** [comment]
@wkeeling Great! Thank you!

**Comment by wkeeling** [comment]
Fixed in version 1.1.0
